### PR TITLE
[Blog Post] 0.36: Headless JS, the Keyboard API, & more

### PR DESCRIPTION
**Pull request by hramos**
This is a draft for an upcoming blog post scheduled for October 24, 2016. Do not merge into `master` before that date.

### CODE REVIEW

File: blog/2016-10-24-whats-new-in-react-native.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+```javascript
+AppRegistry.registerHeadlessTask('SomeTaskName', () => require('SomeTaskName'));
+```
+
+Using Headless JS does require some native Java code to be written in order to allow you to start up the service when needed. Take a look at our new [Headless JS docs](/react-native/docs/headless-js-android.html) to learn more!
+
+## Keyboard Component
+
+Working with the on-screen keyboard is now easier with [`Keyboard`](/react-native/docs/keyboard.html). You can now listen for native keyboard events and react to them. For example, to dismiss the active keyboard, simply call `Keyboard.dimiss()`:
```
Comment (lacker) [inline blog/2016-10-24-whats-new-in-react-native.md]: dimiss -> dismiss

File: blog/2016-10-24-whats-new-in-react-native.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Next, register your task in on `AppRegistry`:
+
+```javascript
+AppRegistry.registerHeadlessTask('SomeTaskName', () => require('SomeTaskName'));
+```
+
+Using Headless JS does require some native Java code to be written in order to allow you to start up the service when needed. Take a look at our new [Headless JS docs](/react-native/docs/headless-js-android.html) to learn more!
+
+## Keyboard Component
```
Comment (lacker) [inline blog/2016-10-24-whats-new-in-react-native.md]: Is this a react component? If not, how about callign this section "The Keyboard API"

File: blog/2016-10-24-whats-new-in-react-native.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+Today we've released [React Native v0.36](https://github.com/facebook/react-native/releases/tag/v0.36.0). You can upgrade to v0.36 by running the following commands in a terminal:
+
+```bash
+$ npm install --save react-native@0.36
+$ react-native upgrade
+```
+
+Read on to learn more about what's new in React Native v0.36.
+
+## Headless JS
```
Comment (hramos) [inline blog/2016-10-24-whats-new-in-react-native.md]: It is, good catch.

File: blog/2016-10-24-whats-new-in-react-native.md (hunk 1 above)
Comment (hramos) [inline blog/2016-10-24-whats-new-in-react-native.md]: Ah, silly mitake.

**Comment by lacker** [comment]
How about naming it something SEO'd, like: "0.36: Headless JS, the Keyboard API, & more"

**Comment by hramos** [comment]
Addressed all feedback. I also moved the upgrade instructions to the bottom of the post as it makes for a nicer preview (the first paragraph is displayed in the blog index and whenever the post is shared on social media).

**Comment by caabernathy** [comment]
How about mentioning Apple TV support (talked about here https://medium.com/@7ynk3r/react-native-apple-tv-today-48beb398a1ab) if it is indeed new in this release.

**Comment by hramos** [comment]
If Apple TV support is ready for use, we should mention it here. I thought it was still a WIP though.

**Comment by hramos** [comment]
We also should make sure Apple TV is properly documented before blogging about it. I vote we write a separate post when docs are ready, it's big enough news on its own.

**Comment by lacker** [comment]
Yeah let's write about Apple TV separately - the stuff in this release is not the final stuff on that.

**Comment by hramos** [comment]
@facebook-github-bot shipit-ninja

**Pull request closed without merging**